src/pages/cli/usage/mock.mdx

```diff
@@ -45,7 +45,9 @@ When defining a schema you can use directives from the GraphQL Transformer in lo
 - [@primaryKey, @index](/cli/graphql/data-modeling)
 - [@hasOne, @hasMany, @belongsTo, @manyToMany](/cli/graphql/data-modeling)
 - [@function](/cli/graphql/custom-business-logic#lambda-function-resolver)
+- [@searchable](/cli/graphql/search-and-result-aggregations/)
 
+### Mocking the Lambda triggers on @model types
```

Comment:
```suggestion
### Mocking the Lambda triggers on `@model` types
```

small nit for consistency https://docs.amplify.aws/cli/usage/mock/#connecting-to-a-mock-model-table